### ISSUE TITLE
Property:title

### ISSUE THREAD

**Issue by mtrekels** [tdwg/cd#57]
|  |  |
| ---- | ---- |
| **Definition** | Short title that summarises the collection objects contained within the Collection Description |
| **Dimension** |  |
| **Existing property** | dc:title |
| **Existing class** |  |
| **Existing property identifier** | http://purl.org/dc/elements/1.1/title |
| **Format** | Text |
| **Required** | Yes |
| **Repeatable** |  |
| **Constraints** |  |
| **Examples** |  |
| **Notes** |  |

**Comment by mswoodburn** [comment]
Not sure this should definitely be a mandatory property. In some use cases (like the DiSSCo CDD use case) where the CD scheme is slicing multiple institutions' collections up by multiple dimensions, you might not want to have to name them all individually. People could auto-generate titles in bulk by concatenating e.g. institution name, CD scheme name and relevant dimensions ('Natural History Museum, London | GeographicLocation | Mycology | Terrestrial: Europe' etc), however that would create redundant and duplicated data which would be difficult to keep in sync with changes to source fields.

**Comment by qgroom** [comment]
I guess I'm thinking of dc:title https://www.dublincore.org/specifications/dublin-core/dcmi-terms/terms/title/
It is quite an important term for being a human readable handle to the collection. It is not difficult for people to provide a title as we would not enforce a format. However, it would be really useful as a minimum element to at least have a title, even if there was very little else.

**Comment by wouteraddink** [comment]
I agree it should not be mandatory. I think "name" would be more clear than "title".  It is basically the human readable version of the identifier, not necessary unique, but useful in eg. working with a dataset in a spreadsheet. Should be only one, and always English (as opposed to alternative labels which could be either of type alias, acronym or translation (name in a different language)

**Comment by rondlg** [comment]
I'd agree to name and mandatory

**Comment by magpiedin** [comment]
added example

**Comment by mswoodburn** [comment]
I've changed the name of this property from 'title' to 'name', and added some more examples. 

Regarding the mandatory debate, we could perhaps make it not mandatory - but add a note along the lines of "There are some applications where a name isn't necessary, for example in a granular and structured breakdown of collections for the automated reporting and visualisation of collection metrics. However, the recommendation is to use this property for any applications where human-readability is an important factor."?

**Comment by ianengelbrecht** [comment]
Just my own feeling about terms is that they should be as descriptive as possible without being too verbose. Would cd:collectionName be a possibility? Just so we don't get confused about what name this specifically refers to?

**Comment by mswoodburn** [comment]
Good suggestion, thanks @ianengelbrecht
